spyder/config/main.py

```diff
@@ -465,13 +455,23 @@
               # ---- In widgets/variableexplorer/aarayeditor.py ----
               'variable_explorer/copy': 'Ctrl+C',
               }),
-            ('color_schemes',
+            ('appearance',
              {
+              'icon_theme': 'spyder 3',
+              # Global Spyder fonts
+              'font/family': MONOSPACE,
+              'font/size': MEDIUM,
+              'font/italic': False,
+              'font/bold': False,
+              'rich_font/family': SANS_SERIF,
+              'rich_font/size': BIG,
+              'rich_font/italic': False,
+              'rich_font/bold': False,
               'ui_theme': 'automatic',
               'names': ['emacs', 'idle', 'monokai', 'pydev', 'scintilla',
                         'spyder', 'spyder/dark', 'zenburn', 'solarized/light',
                         'solarized/dark'],
-              'selected': 'spyder',
+              'selected': 'zenburn',

```

Comment:
Ok, let's change this to 'solarized/dark' until we take a better decision.

Reply:
Just to be sure, 'solarized/dark'? This is a bold move :)

Reply:
I agree; Solarized Dark is an even more "extreme" stylistic choice with a very distinct look. We should go with Spyder Dark as a neutral placeholder for now until we discuss it in another PR.

Reply:
I really dislike `Spyder Dark`, sorry. Besides, this will be quickly changed, so I don't see much of a problem with this.

Reply:
Sorry, but the above two statements appear to be inconsistent with one another:

> I really dislike Spyder Dark, sorry.

> Besides, this will be quickly changed, so I don't see much of a problem with it.

If it will be quickly changed, I don't see why there should be such a problem with keeping our conservative, neutral builtin dark theme as the default, until its replaced with something better as opposed to actively switching to a heavily stylized third-party choice that looks out of place with the rest of the Spyder UI .

Failing that, the alternative is to just keep the default theme set to ``spyder`` for now and change it in the PR that closes #8284 . Then we can stop haggling about it here and discuss it over there where we all agree it belongs.

Reply:
I don't want to keep arguing about this, sorry. @dalthviz, please change this to `solarized/dark`.

Reply:
...
...
...